paddle/optimizer/parameter_optimizer.cc

```diff
@@ -0,0 +1,69 @@
+#include <glog/logging.h>
+#include "adadelta_optimizer.h"
+#include "adagrad_optimizer.h"
+#include "adam_optimizer.h"
+#include "lr_policy.h"
+#include "sgd_optimizer.h"
+
+#include "parameter_optimizer.h"
+
+namespace paddle {
+namespace optimizer {
+
+ParameterOptimizer *ParameterOptimizer::create(
+    const ::std::string &config_proto) {
```

Comment:
in c++, more general create interface would be 
```c++
ParameterOptimizer *ParameterOptimizer::create(const std::string &config_proto) {
return ParameterOptimizer::create(config_proto.c_str());
}
ParameterOptimizer *ParameterOptimizer::create(char *config_proto);
```
because create function always be c style strings. either way in c++.
I never see a byte vector to represent a protobuf.